### ISSUE TITLE
Migrate TFA usage to core keras

### ISSUE THREAD

**Issue by LukeWood** [tensorflow/similarity#294]
SyncBatchNormalization is now a core Keras API;

We can reference: `keras.layers.experimental.SyncBatchNormalization`

https://source.corp.google.com/piper///depot/google3/third_party/py/keras/layers/normalization/batch_normalization.py;l=1089
